less/site.less

```diff
@@ -416,26 +416,7 @@ button.btn, button.cta {
   display: none;
 }
 
-p.success.success.success {
-    margin: @spacer/2;
-    color: @success-text-color;
-    font-weight: 500;
-}
-
-p.failure.failure.failure {
-    margin: @spacer/2;
-    color: @failure-text-color;
-    font-weight: 500;
-}
-
-textarea.hidden-clipboard {
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 2em;
-    height: 2em;
-    padding: 0;
-    border: none;
-    outline: none;
-    box-shadow: none;
+.btn-default-2{
```

Comment:
Looks like something went wrong here during merging? The `p.success`, `p.failure` and `textarea.hidden-clipboard` selectors are needed, so can you add them back?

I also suggest we make the CSS more maintainable by moving the `.btn-default-2` selector into the `less/components/regions.less`, nested appropriately. You should also rename it to `.example-button` so it's more obvious that this styling applies specifically to that one button.

Reply:
Alright